spec/integration/inventory_spec.rb

```diff
@@ -3,569 +3,611 @@
+            resource_type_id = uuid_prefix + '-rt-123'
+            resource_type_name = 'ResourceType'
+
+            record("Inventory/#{secure_context}/inventory_#{x}_#{y}",
+                   { uuid_prefix: uuid_prefix },
+                   'Helpers/generate_some_events_for_websocket') do
+              @client.create_feed new_feed_id
+              ret = @client.create_resource_type new_feed_id, resource_type_id, resource_type_name
+              type_path = ret.path
+
+              # create 3 resources
+              @client.create_resource type_path, id_1, 'My Resource 1', 'version' => 1.0
+              @client.create_resource type_path, id_2, 'My Resource 2', 'version' => 1.1
+              resources_closable.close
+              @client.create_resource type_path, id_3, 'My Resource 3', 'version' => 1.2
+
+              @client.delete_feed new_feed_id
+            end
+
+            # wait for the data
+            sleep 5 if !WebSocketVCR.cassette || WebSocketVCR.cassette.recording?
+            [feed_deleted_closable, resource_type_closable, feeds_closable].each(&:close)
+            expect(new_resource_events[id_1]).not_to be_nil
+            expect(new_resource_events[id_1].properties['version']).to eq(1.0)
+            expect(new_resource_events[id_2]).not_to be_nil
+            expect(new_resource_events[id_2].properties['version']).to eq(1.1)
+            # resource with id_3 should not be among events, because we stopped listening before creating the 3rd one
+            expect(new_resource_events[id_3]).to be_nil
+
+            expect(registered_feed_events[new_feed_id]).not_to be_nil
+            expect(registered_feed_events[new_feed_id].id).to eq(new_feed_id)
+
+            expect(deleted_feed_events[new_feed_id]).not_to be_nil
+            expect(deleted_feed_events[new_feed_id].id).to eq(new_feed_id)
+
+            expect(new_resource_types_events[resource_type_id]).not_to be_nil
+            expect(new_resource_types_events[resource_type_id].id).to eq(resource_type_id)
+            expect(new_resource_types_events[resource_type_id].name).to eq(resource_type_name)
+          end
+        end
 
-        expect(new_resource_types_events[resource_type_id]).not_to be_nil
-        expect(new_resource_types_events[resource_type_id].id).to eq(resource_type_id)
-        expect(new_resource_types_events[resource_type_id].name).to eq(resource_type_name)
+        it 'Should return the version' do
+          data = @client.fetch_version_and_status
+          expect(data).not_to be_nil
+        end
       end
     end
+  end
 
-    it 'Should return the version' do
-      data = @client.fetch_version_and_status
-      expect(data).not_to be_nil
+  describe 'Inventory' do
+    it 'Should list feeds when using SSL without certificate disabling the verify' do
+      # change this to the real credentials when updating the VCR
+      super_secret_username = 'username'
+      super_secret_password = 'password'
```

Comment:
> Would it be possible to take those from the config? Like in here https://github.com/hawkular/hawkular-client-ruby/blob/master/spec/spec_helper.rb#L18:L19 ? I am not sure what is better, I guess the user needs to change those anyway.

Done, I'm using setup_inventory_client now and using record instead of VCR.use_cassette.

>  Btw. why jdoe:password that is pre-created in the dev build doesn't work here?
Updated the cassette to use jdoe/password